flyway-core/src/main/java/org/flywaydb/core/internal/database/cloudspanner/CloudSpannerParser.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2010-2020 Redgate Software Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.flywaydb.core.internal.database.cloudspanner;
+
+import java.util.List;
+import org.flywaydb.core.api.configuration.Configuration;
+import org.flywaydb.core.api.logging.Log;
+import org.flywaydb.core.api.logging.LogFactory;
+import org.flywaydb.core.internal.parser.Parser;
+import org.flywaydb.core.internal.parser.ParsingContext;
+import org.flywaydb.core.internal.parser.Token;
+
+public class CloudSpannerParser  extends Parser {
+    private static final Log LOG = LogFactory.getLog(CloudSpannerParser.class);
+    public CloudSpannerParser(Configuration configuration, ParsingContext parsingContext) {
+        super(configuration,parsingContext, 3);
+    }
+
+    @Override
+    protected char getIdentifierQuote() {
+        return '`';
+    }
+
+    protected char getAlternativeIdentifierQuote() {
+        return '\"';
```

Comment:
Only backticks are allowed for quoting identifiers. Enclosing a literal or identifier in double quotes will make Spanner interpret it as a string literal.